src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -120,6 +123,47 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor('event')) {
+            $dispatcherServiceName = $input->getOption('dispatcher');
+            if (!$this->dispatchers->has($dispatcherServiceName)) {
+                return;
+            }
+
+            $dispatcher = $this->dispatchers->get($dispatcherServiceName);
+            $listeners = $dispatcher->getListeners();
+            $eventNames = array_keys($listeners);
+            $listenersName = [];
+            foreach ($listeners as $listOfListener) {
+                $listenersName = array_merge($listenersName, $listOfListener);
+            }
+            $suggestions->suggestValues(array_merge(
+                $eventNames,
+                $listenersName,
+            ));
```

Comment:
Why listeners are added to the list of suggested event names? The argument get values like `kernel.response` or `Symfony\Component\Mailer\Event\MessageEvent`. I did some tests and listener names are not accepted (ex: `Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse`)

```suggestion
            $suggestions->suggestValues(array_keys($dispatcher->getListeners()));
```